apis/go.mod

```diff
@@ -84,3 +84,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+replace github.com/openstack-k8s-operators/nova-operator/api => github.com/gibizer/nova-operator/api v0.0.0-20230621113659-2d71d685e22e
```

Comment:
Looks like this is having trouble passing CI because of this.  Perhaps we can just wait to re-run the tests until the Nova patch lands (and update this patch to remove this line).

Reply:
Yeah it seems it won't pass until that. I think on of the complication is that my github ID is gibizer while my quay.io user is gibi :/

Reply:
Anyhow I'm trying to run test manually with nova- and openstack-operator changes together and report back here.

Reply:
I was able to test it and the validation webhook call to the nova-operator works:

```
$OPENSTACK_IMG=quay.io/gibi/openstack-operator-index:v0.0.114 make openstack
[snip]
$ make openstack_deploy
[snip]
$ oc patch -n openstack OpenStackControlPlane/openstack-network-isolation --type=merge -p '{"spec":{"nova":{"template": {"cellTemplates": {"cell0": {"metadataServiceTemplate": {"replicas": 1}}}}}}}'
The OpenStackControlPlane "openstack-network-isolation" is invalid: spec.nova.template.cellTemplates[cell0].metadataServiceTemplate.replicas: Invalid value: 1: should be 0 for cell0
```
And in the openstack-operator logs:
```
{"level":"info","ts":"2023-06-22T16:52:10.852Z","logger":"openstackcontrolplane-resource","msg":"default","name":"openstack-network-isolation"}
{"level":"info","ts":"2023-06-22T16:52:10.861Z","logger":"openstackcontrolplane-resource","msg":"validate update","name":"openstack-network-isolation"}
```

Reply:
https://github.com/openstack-k8s-operators/nova-operator/pull/411 has landed.  Let's retry the CI once we remove the fork `replace`.